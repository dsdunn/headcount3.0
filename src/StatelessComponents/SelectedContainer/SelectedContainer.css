.selected-container {
  display: flex;
  min-height:1px;
  justify-content: space-around;
}
.selected-container .card {
  box-shadow: 7px 5px #A6A29B;
  transform: scale(1.05);
  transition: all .2s ease-in-out;
  font-size: 1em;
  border-width: 2px;
  border-color: #FF8F06;
}