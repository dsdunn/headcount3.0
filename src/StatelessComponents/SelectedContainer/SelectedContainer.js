import React from 'react';
import Card from '../Card/Card';

const SelectedContainer = ({ selectedCards }) => {
  return (
      
    )
}

export default SelectedContainer;