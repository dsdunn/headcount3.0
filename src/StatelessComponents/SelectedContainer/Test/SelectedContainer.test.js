import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import SelectedContainer from '../SelectedContainer';

describe('SelectedContainer', () => {
  let wrapper = shallow(<SelectedContainer selectedCards={[]}/>)
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})