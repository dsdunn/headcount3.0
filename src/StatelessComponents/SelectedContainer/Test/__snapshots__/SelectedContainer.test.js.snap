// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectedContainer should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SelectedContainer
    selectedCards={Array []}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        undefined,
        false,
        undefined,
      ],
      "className": "selected-container",
    },
    "ref": null,
    "rendered": Array [
      undefined,
      false,
      undefined,
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          undefined,
          false,
          undefined,
        ],
        "className": "selected-container",
      },
      "ref": null,
      "rendered": Array [
        undefined,
        false,
        undefined,
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
